### PR TITLE
Fix documentation builds and pull in version picker

### PR DESCRIPTION
**Pull request by Half-Shot**
This largely cribs off the hookshot workflows, and blatantly steals the version picker code.

### CODE REVIEW

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -1,43 +0,0 @@
-name: Build docs
-
-on:
-  push:
-    branches:
-      - master
```
Comment (Half-Shot) [inline .github/workflows/docs.yml:6]: This was broken because we don't use `master` anymore.

File: .github/workflows/docker-hub-latest.yml [hunk 2]
```diff
@@ -4,7 +4,8 @@ name: "Docker Hub - Latest"
 
 on:
   push:
```
Comment (Half-Shot) [inline .github/workflows/docker-hub-latest.yml:6]: I believe push on it's own excludes forks, which we don't want to do.

**Comment by AndrewFerr** [comment]
Are the E2E test failures the same ones caused by Redis sending an error on shutdown?

**Comment by Half-Shot** [comment]
This flaked but I'm not sure why :/

**Pull request merged**